src/main/java/com/example/repick/domain/clothingSales/repository/ClothingSalesStateRepository.java

```diff
@@ -0,0 +1,15 @@
+package com.example.repick.domain.clothingSales.repository;
+
+import com.example.repick.domain.clothingSales.entity.ClothingSalesState;
+import com.example.repick.domain.clothingSales.entity.ClothingSalesStateType;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+public interface ClothingSalesStateRepository extends JpaRepository<ClothingSalesState, Long>{
+    @Query("SELECT MAX(c.createdDate) FROM ClothingSalesState c WHERE c.clothingSalesId = :clothingSalesId AND c.clothingSalesStateType = :clothingSalesStateType")
+    Optional<LocalDateTime> findLatestCreatedDateByClothingSalesIdAndStateType(@Param("clothingSalesId") Long clothingSalesId, @Param("clothingSalesStateType") ClothingSalesStateType clothingSalesStateType);
+}
```

Comment:
```suggestion
package com.example.repick.domain.clothingSales.repository;

import com.example.repick.domain.clothingSales.entity.ClothingSalesState;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ClothingSalesStateRepository extends JpaRepository<ClothingSalesState, Long>{
//    @Query("SELECT MAX(c.createdDate) FROM ClothingSalesState c WHERE c.clothingSalesId = :clothingSalesId AND c.clothingSalesStateType = :clothingSalesStateType")
//    Optional<LocalDateTime> findLatestCreatedDateByClothingSalesIdAndStateType(@Param("clothingSalesId") Long clothingSalesId, @Param("clothingSalesStateType") ClothingSalesStateType clothingSalesStateType);

    List<ClothingSalesState> findAllByClothingSalesId(Long id);
}
```

아까 제안 관련한 findAllByClothingSalesId 구현 부분임니다